libsplinter/src/migrations/diesel/postgres/migrations/2020-11-24-125000_admin_service_mailbox/up.sql

```diff
@@ -0,0 +1,19 @@
+---- Copyright 2018-2020 Cargill Incorporated
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- -----------------------------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS event_entry (
```

Comment:
Thoughts on prefixing with admin so `admin_event_entry`. We may want to add event entry information in the future that is about a different component and this would avoid confusion/conflicts.